portfolio/src/main/webapp/script/comment-script.js

```diff
@@ -146,27 +147,33 @@ function createImageDiv(imageUrl) {
  * Creates and returns a <div> element containing {@code text}, {@code email},
  * and {@code timeStamp} from comment.
  */
-function createCommentDiv(text, email, timeStamp) {
+function createCommentDiv(text, email, timeStamp, sentimentScore) {
   const commentOuterDiv = document.createElement('div');
   const commentDiv = document.createElement('div');
 
   const textElement = document.createElement('p');
   const emailElement = document.createElement('h4');
+  const scoreElement = document.createElement('h5');
   const dateElement = document.createElement('h5');
 
+  var scoreWord = sentimentScore >= 0 ? 'good' : 'bad';
```

Comment:
Please add a comment on what >=0 means and <0 means? What does good or bad mean, its quite vague and a bit of an explanation would help.

Reply:
fixed